docs/API.md

```diff
@@ -3,47 +3,59 @@
 The current live api lives at https://api.jujucharms.com/charmstore/v4
 
 ## Intro
-The charm store stores and indexes charms and bundles. A charm or bundle is referred to by a charm store id which can take one of the following two forms:
+
+The charm store stores and indexes charms and bundles. A charm or bundle is
+referred to by a charm store id which can take one of the following two forms:
 
 * ~owner/series/name(-revision)
 * series/name(-revision)
 
 *Owner* is the name of the user that owns the charm.
-*Series* is one of a small number of known possible series for charms (currently just the Ubuntu series names) or the special name bundle to signify that the charm id refers to a charm bundle.
+*Series* is one of a small number of known possible series for charms
+(currently just the Ubuntu series names) or the special name bundle to signify
```

Comment:
@rogpeppe sorry you're right. We've not yet updated to the flat namespace ids on the charmstore api.